### PR TITLE
Sample for apache-php7

### PR DESCRIPTION
**Pull request by kouul**

### CODE REVIEW

File: apache-php/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+```
+
+[_docker-compose.yaml_](docker-compose.yaml)
+```
+services:
+  web:
+    build: app
+    ports: 
+      - '8008:80'
```
Comment (glours) [inline apache-php/README.md:20]: Can we keep the default port of Apache?

File: apache-php/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+version: "3.7"
+services:
+  web:
+    build: app
+    ports: 
+      - '8008:80'
```
Comment (glours) [inline apache-php/docker-compose.yaml:6]: Can we keep the default port of Apache?

File: apache-php/docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    volumes:
+      - ./app:/var/www/html/
+
```
Comment (glours) [inline apache-php/docker-compose.yaml:9]: nit: add a new line at  the end of the file

File: apache-php/README.md (hunk 1 above)
Comment (kouul) [reply apache-php/README.md]: done

File: apache-php/docker-compose.yaml (hunk 2 above)
Comment (kouul) [reply apache-php/docker-compose.yaml]: done

File: apache-php/docker-compose.yaml (hunk 3 above)
Comment (kouul) [reply apache-php/docker-compose.yaml]: done

File: apache-php/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+```
+
+## Expected result
+
+Listing containers must show two containers running and the port mapping as below:
+```
+$ docker ps
+CONTAINER ID        IMAGE                        COMMAND                  CREATED             STATUS              PORTS                  NAMES
+2bc8271fee81        php-docker_web               "docker-php-entrypoi…"   About a minute ago  Up About a minute   0.0.0.0:8008->80/tc    php-docker_web_1
```
Comment (glours) [inline apache-php/README.md:44]: You forget to update the port in the docker ps description ;-)

File: apache-php/README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+After the application starts, navigate to `http://localhost:8008` in your web browser or run:
```
Comment (glours) [inline apache-php/README.md:47]: same here

File: apache-php/README.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+```
+$ curl localhost:8008
```
Comment (glours) [inline apache-php/README.md:49]: and here

Comment (kouul) [reply apache-php/README.md]: Oops my bad.. its fixed :+1:

**Pull request merged**